docs/references/qa/CometBFT-QA-v1.md

```diff
@@ -0,0 +1,302 @@
+transaction rate ranging from 100 to 1000 tx/s, in intervals of 100.
+
+![v1_saturation](imgs/v1/saturation/saturation_v1_LE.png) 
+
+These are the actual values which we use to generate the figure:
+
+| r    | v1 without LE | v1 with LE   | 
+| ---: | ----: | ----: |
+| 100  |  8900 |  8900 |
+| 200  | 17800 | 17800 |
+| 300  | 26053 | 26700 |
+| 400  | 28800 | 35600 |
+| 500  | 32513 | 34504 |
+| 600  | 30455 | 42169 |
+| 700  | 33077 | 38916 |
+| 800  | 32191 | 38004 |
+| 900  | 30688 | 34332 |
+| 1000 | 32395 | 36948 |
+
+## 200-nodes test
+
+This experiment consist in running the 200-nodes network, injecting a load of 400 tx/s (`c=1,r=400`)
+during 90 seconds, and collecting the metrics. The network composition is the same as used for
+finding the saturation point.
+
+For the experiments with latency emulation have set a duration of 180 seconds instead of the 90.
+
+### Latencies
+
+The following figures show the latencies of the experiment. Each dot represents a block: at which
+time it was created (x axis) and the average latency of its transactions (y axis).
+
+| v0.38 | v1 (without LE / with LE) 
+|:--------------:|:--------------:|
+| ![latency-1-400-v38](img38/200nodes/e_de676ecf-038e-443f-a26a-27915f29e312.png) | ![latency-1-400-v1](imgs/v1/200nodes/latencies/e_8e4e1e81-c171-4879-b86f-bce96ee2e861.png) 
+| | ![latency-1-400-v1-le](imgs/v1/200nodes_with_latency_emulation/latencies/e_8190e83a-9135-444b-92fb-4efaeaaf2b52.png)
+
+In both cases, most latencies are around or below 4 seconds. On `v0.38` there are peaks reaching 10
+seconds, while on `v1` (without LE) the only peak reaches 8 seconds. In general, the images are
+similar. From this small experiment we infer that the latencies measured on `v1` are not worse than
+those of the baseline. With latency emulation, the latencies are considerably higher, as expected.
+
+### Metrics
+
+In this section we analyse key metrics extracted from Prometheus data on the 200-nodes experiment
+with configuration `c=1,r=400`.
+
+#### Mempool size
+
+The following figures show the evolution of the average and maximum mempool size over all full
+nodes. 
+
+**Average size** On `v1`, the average mempool size mostly stays below 1000 outstanding transactions
+except for a peak above 2000, coinciding with the moment the system reached round number 1 (see
+below). For these particular runs, this result is better than the baseline, which oscilates between
+1000 and 2500.
+
+With latency emulation, the average mempool size stays mostly above 2000 outstanding transactions
+with peaks almost reaching the maximum mempool size of 5000 transactions.
+
+| v0.38 | v1 (without LE / with LE) 
+| :--------------:|:--------------:|
+| ![mempool-avg-baseline](img38/200nodes/avg_mempool_size.png) | ![mempool-avg](imgs/v1/200nodes/metrics/avg_mempool_size.png)
+| | ![mempool-avg-le](imgs/v1/200nodes_with_latency_emulation/metrics/avg_mempool_size.png)
```

Comment:
Nitpicking: Would it be possible to use the same y-axis range across these 3 pictures? This will make the plots comparable visually. Maybe use the range 0 - 5000, for example. 

Nice to have, so if it's difficult to achieve nevermind.

Reply:
I regenerated all images to have the same range in the y axis. Not that simple with our current setup but it was needed.